# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../prewikka/setup_py.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./setup.py.orig	2007-08-04 01:02:10.000000000 +0200
+++ ./setup.py	2007-08-04 01:02:55.000000000 +0200
@@ -183,7 +183,6 @@
     def run(self):
         os.umask(022)
         self.install_conf()
-        self.init_siteconfig()
         install.run(self)
         
         for dir in ("/",
